src/main/java/frc/robot/util/Node.java

```diff
@@ -54,48 +54,38 @@ public Node(Vision vision, Alliance alliance, int row, int column) {
           : NodeType.CONE;
 
     // Starting locations
-    double x = (alliance == Alliance.Blue ? 
-      FieldConstants.kBlueAllianceNodeStartX : 
-      FieldConstants.kRedAllianceNodeStartX
-    );
+    double x = 0;
+    if (alliance == Alliance.Blue) {
+      x = VisionConstants.kAprilTags.get(5).pose.getX() + FieldConstants.kAprilTagOffset + FieldConstants.kRobotDistanceFromNodeTape;
```

Comment:
AprilTag 5 is on the double substation, I assume you want an AprilTag on the grid